custom-stats-logger/src/main/java/dev/gemfire/stat/logger/StatLogger.java

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023-2024 Broadcom. All rights reserved.
+
+package dev.gemfire.stat.logger;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.geode.StatisticDescriptor;
+import org.apache.geode.Statistics;
+import org.apache.geode.StatisticsType;
+import org.apache.geode.cache.*;
+import org.apache.geode.cache.execute.Function;
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.distributed.DistributedSystem;
+import org.apache.geode.logging.internal.log4j.api.LogService;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class StatLogger implements Function,Declarable {
+  private static final Logger logger = LogService.getLogger();
+  private List<StatsHolder> listOfStats = new CopyOnWriteArrayList<>();
+  private Timer timer;
+  private TimerTask task;
+  private long timerInterval = 5000;
+
+  @Override
+  public void initialize(Cache cache, Properties props) {
+    logger.info("Initializing Stat Logger with properties {}", props);
```

Comment:
Is the code actually being executed? I don't see this method actually doing anything with the properties other than logging them.
execute has its own way of getting the properties so should you just let it take care of doing logging if it is needed? I'm not sure it is since it reports back in its result what it has done based on the properties it reads.

Reply:
Yeah I have removed that Darrel. That was a stale code from something I was trying out.